### ISSUE TITLE
Pagination not SEO friendly there is no way to set "href" on a tags auto created.

### ISSUE THREAD

**Issue by xaander1** [Semantic-Org/Semantic-UI-React#4376]
I think maybe there should be a way to make pagination SEO friendly at least.

**Comment by riosant** [comment]
I agree with @xaander1 There should be something where we can define what to render in each number of pagination. I am working with nextjs and the semantic pagination component doesn't have href attribute in pagination elements.
